test/markup/lsl/hello-avatar.expect.txt

```diff
@@ -0,0 +1,12 @@
+<span class="hljs-keyword">default</span>
+{
+    <span class="hljs-event">state_entry</span>()
+    {
+        <span class="hljs-function">llSay</span>(<span class="hljs-constant.integer">PUBLIC_CHANNEL</span>, "Hello, Avatar!");
+    }
+
+    <span class="hljs-event">touch_start</span>(<span class="hljs-type">integer</span> num_detected)
+    {
+        <span class="hljs-function">llSay</span>(<span class="hljs-constant.integer">PUBLIC_CHANNEL</span>, "Touched.");
+    }
+}
```

Comment:
Markup tests are intended to test specific features of the language. If it simply duplicates the detection test, it's not needed.